src/renderers/shared/hooks/__tests__/ReactComponentTreeHook-test.js

```diff
@@ -185,7 +185,7 @@ describe('ReactComponentTreeHook', () => {
       assertTreeMatches([element, tree]);
     });
 
-    it('uses displayName, name, or Object for factory components', () => {
+    xit('uses displayName, name, or Object for factory components', () => {
```

Comment:
This is important to document. Also many people will try to use `it.only`, it will crash, and people will think Jest doesn't support this feature.

@cpojer